### DISCUSSION TITLE
2. Which datasets / resources in our repository are you still using ?

### DISCUSSION THREAD

**Discussion by andreasamsler** [openZH/covid_19#1479] [General]
Name the datasets / resources, and - optionally - briefly explain how you are using them (e.g.):
- as the main data source (at all and/or to fill which gaps of a certain other data source) ?
- as a second source (to check for plausibility for instance) ?
- on a daily basis or less frequently ?
- anything else, which could be relevant to assess the need for the datasets / ressources ?

**Comment by dkoltg** [comment]
We use numbers for the positiv tested persons

**Comment by fabian** [comment]
Just noticed this, looks like the Swiss National COVID-19 Science Task Force (no affiliation) is using some of the data as additional data points. [Lagebeurteilung, 14. Dezember 2020](https://sciencetaskforce.ch/epidemiologische-lagebeurteilung-14-dezember-2020/) contains the following:

> Seit dem 1. Oktober 2020 weist das Bundesamt für Gesundheit 3’769 laborbestätigte Todesfälle aus.<sup>5</sup> Die Kantone meldeten in dieser Zeit 4’158 Todesfälle.<sup>6</sup>
> [5] https://www.covid19.admin.ch
> [6] https://github.com/openZH/covid_19 und https://github.com/daenuprobst/covid19-cases-switzerland

**Comment by one-github** [comment]
- as the main data source (at all and/or to fill which gaps of a certain other data source) ?
Fill gaps with test positivity found in R(eff) calculations of the Swiss Science Task Force.  I use these files of yours:
covid_19/fallzahlen_tests/fallzahlen_kanton_*_tests.csv

- on a daily basis or less frequently ?
I push changes to Twitter (see https://twitter.com/SwissC19SitRep) as they happen on your repo.

- anything else, which could be relevant to assess the need for the datasets / ressources ?
Github is a great place for this for many reasons. Thank you for your work!

**Comment by d-qn** [comment]
We use the number of currently hospitalised covid patients and current ICU patients. 

- As the main source
- On a daily basis (or more often, whenever there is an update)
- I haven't switched to BAG API data for that yet, because I am not sure of the difference/similarity between your and their dataset

**Reply by amslera** [reply]
Regarding the data on 'current_hosp' and 'current_icu' from the FOPH resp. from the IES system of the KSD:
- Data: https://www.covid19.admin.ch/api/data/20201215-rfotj1y8/sources/COVID19HospCapacity_geoRegion.csv
(🙏 retrieving the data requires some knowledge, can sb write it down in a comment?)
- @rsalzer visualised the data https://twitter.com/rob_salzer/status/1338983660461174787?s=21, they are differentiated by Covid19 and all other diagnoses, and are available for all cantons individually.
- There may be discrepancies between the values provided by the FOPH and those communicated by the cantons on the web.
- In the case of ZH, where the Department of Health provides values at hospital level as OGD, these deviations are low and it is plausible that the reason lies in the different times of the survey and the publication.

👉 I recommend comparing both sources for two or three days, waiting for the weekend and then deciding to switch to the FOPH source. 
👉 I expect this source to be more stable over the festive season and the turn of the year than the communication flow of the cantons and thus the data of the cantons in our repo.

**Reply by metaodi** [reply]
This is the API link that provides the current valid URLs to the BAG data: https://www.covid19.admin.ch/api/data/context

Using this, it is straightforward to automate the download of the datasets either as CSV or JSON.

**Reply by amslera** [reply]
In addition to the bullet point „In the case of ZH, ...“ above mentioned:
- the ICU related data communicated and available as OGD by the Department of Health of the Canton of Zurich (GD-ZH) is collected via an information system called „rescue track“, which is in service since March 2020 and is used for the disposition of rescue vehicles, too.
- reasons for deviations to the data collected via the IES system of KSD/SANKO and available by the FOPH are:
1. differing times of data entry/collection and
2. differing definitions between the two systems.

GD-ZH is continuing to deliver the data from their „rescue track“ system, on the basis expressed that: 
- the data is more up to date
- definitions are more appropriate (e.g. GD-ZH claims there are no ‚Covid beds‘ and ‚Non-Covid beds‘, as collected in the IES system).

**Reply by rsalzer** [reply]
> Regarding the data on 'current_hosp' and 'current_icu' from the FOPH resp. from the IES system of the KSD:
> 
> * There may be discrepancies between the values provided by the FOPH and those communicated by the cantons on the web.

The data seems to be the same as on [icumonitoring.ch](http://www.icumonitoring.ch) but not "certified" beds but "ad-hoc" ones.
Unfortunately the description of the data itself is not clear about that: https://www.covid19.admin.ch/api/data/documentation/models/sources-definitions-hospcapacitydailyincomingdata.md

**Comment by balzamas** [comment]
Name the datasets / resources, and - optionally - briefly explain how you are using them (e.g.):

As the main data source: 
- fallzahlen_bezirke (all of them, except SZ and AG)
- COVID19_Fallzahlen_CH_total_v2.csv

All: daily

I read the data into my django and use it in the form of tables, graphics and maps.

**Comment by amslera** [comment]
https://neherlab.org, a research group at the Biozentrum, University of Basel is using (https://github.com/neherlab/covid19_scenarios/blob/master/data/parsers/switzerland.py#L86):
- cases
- hospitalized
- ICU
- deaths

for https://covid19-scenarios.org/

**Reply by amslera** [reply]
Neherlab was informed on 2020-12-16 about the IES data source made available by FOPH and KSD/SANKO, which is delivering HOSP and ICU related data *differentiated by Covid19 and all other diagnoses*, and *for all cantons individually*.

More: https://github.com/openZH/covid_19/discussions/1479#discussioncomment-213049

**Comment by rsalzer** [comment]
I am using:

For https://rsalzer.github.io/COVID_19_CH:
- COVID19_Fallzahlen_CH_total_v2.csv

For https://rsalzer.github.io/COVID_19_KT_ZH:
- COVID19_Fallzahlen_Kanton_ZH_total.csv
- fallzahlen_bezirke/fallzahlen_kanton_ZH_bezirk.csv
- fallzahlen_plz/fallzahlen_kanton_ZH_plz.csv
- fallzahlen_kanton_zh/COVID19_Belegung_Intensivpflege.csv
- Plus the provided GeoJSONs

**Comment by jlsmdl** [comment]
Since today we at SRF are not using any datasets from your repository any more.
